CRM/Price/BAO/PriceFieldValue.php

```diff
@@ -39,7 +39,7 @@ public static function add($params) {
     }
 
     // Reset the cached values in this function.
-    CRM_Price_BAO_PriceField::getOptions(CRM_Utils_Array::value('price_field_id', $params), FALSE, TRUE);
+    CRM_Price_BAO_PriceField::getOptions(0, FALSE, TRUE);
```

Comment:
This function was only being called to reset its cache, so the first param wasn't even being used.